jobs/scripts/glusto/run-glusto-main-all.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Run ansible script to setup everything
+# Get master IP
+host=$(grep ansible_host hosts | head -n 1 | awk '{split($2, a, "="); print a[2]}')
+
+
+set -x
+ 
+GLUSTO_WORKSPACE="$WORKSPACE"
+# todo: not sure if this setup needed 
+if [ -d "glusto" ]; then
+  mv hosts centosci/hosts
```

Comment:
im not sure if this lines needed, 
Its running for glusto-patch-check in case glusto/ dir exists so i kept them

Reply:
Yup, we are not looking for glusto run on top of any patch.So we can skip those lines.